src/components/output/index.tsx

```diff
@@ -39,13 +39,17 @@ export default class Output extends Component<Props, State> {
     }
   }
 
+  shouldComponentUpdate(nextProps: Props) {
+    return this.props.leftImg !== nextProps.leftImg || this.props.rightImg !== nextProps.rightImg;
+  }
```

Comment:
I did this to reduce calls to render() while I was debugging. Is it worth keeping around?